src/test/java/org/voltdb/exportclient/TestRabbitMQExportClient.java

```diff
@@ -117,9 +129,9 @@ public void testPartitionColumnEffectiveRoutingKey() throws Exception
                          /* expectedRoutingKey */ "yankeelover.4");
 
         // Replicated table, no routing suffix by default
-        verifyRoutingKey(/* replicated */ true,
-                         /* colName */ null,
-                         /* expectedRoutingKey */ "yankeelover");
+//        verifyRoutingKey(/* replicated */ true,
```

Comment:
If table is replicated and no partition column specified, this test condition is not true any more. The test logic in here expected the routing key to be just table name for this case but it isn't. The behavior is same as on master.